velox/exec/tests/TableScanTest.cpp

```diff
@@ -1735,6 +1735,48 @@ TEST_P(TableScanTest, structLazy) {
   assertQuery(op, {filePath}, "select c0 % 3 from tmp");
 }
 
+TEST_P(TableScanTest, structInMap) {
+  vector_size_t size = 1'000;
+
+  auto rowNumbers = makeFlatVector<int64_t>(size, [](auto row) { return row; });
+  auto innerRow = vectorMaker_.rowVector({rowNumbers});
```

Comment:
nit: do not use vectorMaker_ directly; instead use makeRowVector method from the base class